.github/workflows/integration-tests-logcollector-tier-0-1-lin.yml

```
@@ -0,0 +1,85 @@
+name: Integration tests for logcollector on Linux - Tier 0 and 1
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch'
+        required: true
+        default: 'main'
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-logcollector-tier-0-1-lin.yml"
+        - "src/logcollector/**"
```

Add also config folder in all three workflows.
```suggestion
        - "src/config/localfile-config.c"
        - "src/config/localfile-config.h"
        - "src/logcollector/**"
```

---

Done!